demo/app/frontend/packs/application.scss

```diff
@@ -1,5 +1 @@
 @import '~@citizensadvice/design-system/scss/lib.scss';
-
-#content {
```

Comment:
I removed this as for header and footer component previews it helps for them to be flush. Will revisit this in a future PR as it would be nice to specific either for previews.

Reply:
No probs - FYI I added it for the field component as it needs to not be flush in the error state.